src/assertions/assertions.csproj

```diff
@@ -23,4 +24,9 @@
         <PackageReference Include="Neo" Version="$(NeoVersion)" />
     </ItemGroup>
 
+
+    <ItemGroup>
+      <PackageReference Update="Nerdbank.GitVersioning" Version="3.6.139" />
```

Comment:
`Directory.Build.props` handles this too.

Reply:
fixed

Reply:
you can update version here https://github.com/neo-project/neo-express/blob/master/Directory.Build.props if you haven't already did so.